tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -25,15 +25,11 @@ sub run {
 
     if (check_var('RUN_TEST_ONLY', 0)) {
         use_ssh_serial_console;
-        my $sut_ip = get_required_var('SUT_IP');
-
-        set_var('AGENT_IP', $sut_ip);
-        bmwqemu::save_vars();
 
         # Synchronize the server & agent node before setup
         barrier_wait('kubevirt_test_setup');
 
-        my $server_ip = $self->get_var_from_parent("SERVER_IP");
+        my $server_ip = $self->get_var_from_parent("SUT_IP");
         record_info('Server IP', $server_ip);
```

Comment:
You still refer to server ip here.

Reply:
Actually, it will get the value of SUT_IP from master node, depends on which one is the parent machine.